sdk/push-fcm/src/main/java/com/klaviyo/pushFcm/KlaviyoNotification.kt

```diff
@@ -114,7 +129,7 @@ class KlaviyoNotification(private val message: RemoteMessage) {
      * @param context
      * @return [Notification] to display
      */
-    private fun buildNotification(context: Context): Notification =
+    private fun buildNotification(context: Context): NotificationCompat.Builder =
```

Comment:
nit: I feel like this kinda changes the docstring tied to the function too. We're no longer returning a `Notification` instance but a builder

Reply:
oh most definitely